modules/javafx.web/src/test/java/test/javafx/scene/web/EventListenerLeakTest.java

```diff
@@ -627,13 +649,370 @@ public void TestStrongRefNewContentLoad() throws Exception {
         // load new content
         loadContent(webView1, HTML2);
 
-        // Verify that all listeners have not been released
-        Thread.sleep(100);
         submit(() -> {
             // Send click event
             click(webView1, 0);
         });
 
+        // Verify that all listeners have not been released
+        Thread.sleep(100);
         assertEquals("Click count", 1, listeners1.get(0).getClickCount());
+
+        // Clear strong reference to listener and WebView
+        listeners1.clear();
+        domNodes1.clear();
+        webViewRefs.clear();
+        webView1 = null;
+
+        // Verify that there is no strong reference to the WebView
+        assertNumActive("WebView", webViewRefs, 0);
+
+        // Verify that no listeners are strongly held
+        assertNumActive("MyListener", listenerRefs, 0);
+        listenerRefs.clear();
```

Comment:
It is not necessary to clear the `listenerRefs` list. There is never a need for a test to explicitly modify this list.